app/src/pages/organization/projectTeamPage/projectTeamListTable/projectTeamListTable.scss

```diff
@@ -71,3 +71,15 @@
   align-items: center;
   height: 100%;
 }
+
+.with-pagination-container {
+  position: relative;
+}
+
+.with-pagination {
```

Comment:
I see that you completely duplicate these styles.
Why not to create a small wrapping component around `Pagination` and set the styles there together with `captions` to avoid unnecessary prop drilling and code duplication?

Reply:
Done.